<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="qgb_game">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Игра</string>
    </property>
    <widget class="QPushButton" name="qpb_game_end">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>150</y>
       <width>441</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Завершить игру</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qpb_game_move">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>120</y>
       <width>441</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Сделать ход</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Ваш ход:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Палочки:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ходит:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ql_game_player_now">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Игрок 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="ql_game_sticks_now">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>371</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>||||| (5)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="qsb_game_step">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>371</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="qgb_settings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры игры</string>
    </property>
    <widget class="QGroupBox" name="qgb_mod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>441</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Режим игры</string>
     </property>
     <widget class="QRadioButton" name="qrb_human_computer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>210</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Пользователь - Компьютер</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="qrb_human_human">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Пользователь - Пользователь</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="qpb_game_begim">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>441</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Начать игру</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Первый ход </string>
     </property>
    </widget>
    <widget class="QComboBox" name="qcb_game_player_first">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Случайно</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Игрок 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Игрок 2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество палочек </string>
     </property>
    </widget>
    <widget class="QPushButton" name="qpb_game_sticks_random">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Случайно</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="qsb_game_sticks">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
